fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -83,17 +82,16 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
 
 @click.command()
 @click.pass_context
-def ping(ctx: click.Context) -> None:
+def status(ctx: click.Context) -> None:
     """
     Sends a request to the Fidesctl API healthcheck endpoint and prints the response.
     """
     config = ctx.obj["CONFIG"]
-    healthcheck_url = config.cli.server_url + "/health"
-    echo_green(f"Pinging {healthcheck_url}...")
-    try:
-        handle_cli_response(with_analytics(ctx, _api.ping, url=healthcheck_url))
-    except requests.exceptions.ConnectionError:
-        echo_red("Connection failed, webserver is unreachable.")
+    cli_version = fidesctl.__version__
+    server_url = config.cli.server_url
+    echo_green("Getting server status...")
+    check_server(cli_version, server_url)
+    echo_green("Server is reachable and the client/server application versions match.")
```

Comment:
As written, `check_server` may `echo_red` that there's a version mismatch, but `status` would still `echo_green` that the versions do match. I'd prefer if `check_server` also raise an exception in the case of a mismatch, and all logging be handled here (at the highest level possible) using `try`/`except` logic. Something simple like:

```python
try:
	check_server(cli_version, server_url)
except requests.exceptions.ConnectionError as err:
	echo_red("oh dear")
	raise err
except VersionMismatchException as err:
	echo_red("oy vey")
	raise err
else:
	echo_green("all good")
```

Reply:
updated to do fail if versions don't match, not quite ready to jump into custom error territory yet, we can use the simple tools we already have to accomplish the same thing for now